pkg/util/apply/pdb.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2021 ScyllaDB
+
+package apply
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	"k8s.io/api/policy/v1beta1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/record"
+)
+
+// PodDisruptionBudget creates or updates PDB. If desired object is equal (based on calculated hash)
+// to existing one update is not called.
+func PodDisruptionBudget(ctx context.Context, recorder record.EventRecorder, client kubernetes.Interface, pdb *v1beta1.PodDisruptionBudget) (*v1beta1.PodDisruptionBudget, error) {
```

Comment:
Would you mind wiring it using listers (caches)? No expectations, just easy erroring out until informers catch up.

Reply:
It should also merge metadata to ensure it has e.g. correct labels

Reply:
actually, hashing the whole object is sufficient for PDB